pipeline/outputs/stackdriver.md

```diff
@@ -15,6 +15,7 @@ Before to get started with the plugin configuration, make sure to obtain the pro
 | google\_service\_credentials | Absolute path to a Google Cloud credentials JSON file | Value of environment variable _$GOOGLE\_SERVICE\_CREDENTIALS_ |
 | service\_account\_email | Account email associated to the service. Only available if **no credentials file** has been provided. | Value of environment variable _$SERVICE\_ACCOUNT\_EMAIL_ |
 | service\_account\_secret | Private key content associated with the service account. Only available if **no credentials file** has been provided. | Value of environment variable _$SERVICE\_ACCOUNT\_SECRET_ |
+| metadata\_server | Prefix for a metadata server. Can also set environment variable _$METADATA\_SERVER_  | http://metadata.google.internal |
```

Comment:
When the description is multiple sentences, there's a trailing period.

Reply:
Fixed.